### PR TITLE
Add multi-lang components blog post

### PR DESCRIPTION
**Pull request by lblackstone**

### CODE REVIEW

File: content/blog/multi-lang-components/index.md [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+tags:
+    - eks
+    - .net
+    - python
+    - go
+    - typescript
+---
+
+Today we're excited to announce support for one of our most highly-requested features: multi-language components! Our
+first multi-language component is the popular EKS package, which was previously only available for TypeScript. With the
```
Comment (stack72) [inline content/blog/multi-lang-components/index.md:27]: Needs a link to the eks package

File: content/blog/multi-lang-components/index.md [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+[v0.21.0 release] (TODO link), teams can manage EKS clusters in their language of choice (Python, .NET, Go, and TypeScript)!
```
Comment (stack72) [inline content/blog/multi-lang-components/index.md:28]: I don't think dotnet and Go will be in the v0.21 release will it? There's only Python setup in place at this point in our builds

File: content/blog/multi-lang-components/index.md [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+
+<!--more-->
+
+Pulumi's support for many familiar programming languages means that your organization is free to choose which language
```
Comment (stack72) [inline content/blog/multi-lang-components/index.md:32]: Within a list of what we support - it's not open to anything

File: content/blog/multi-lang-components/index.md [hunk 4]
```diff
@@ -0,0 +1,163 @@
...
+works best for each team. It's already possible to collaborate across languages by using separate stacks and referring to
```
Comment (stack72) [inline content/blog/multi-lang-components/index.md:33]: Projects can change languages - stacks can't - right? I mean dev1 stack and dev2 stack need to be the same Lang

**Review by stack72 — COMMENTED** [review]
Initial thoughts only

File: content/blog/multi-lang-components/index.md [hunk 5]
```diff
@@ -0,0 +1,163 @@
...
+them using Stack References (TODO link). Multi-language components take this one step further towards the vision of
+"write once, use anywhere". Component authors broaden their reach across languages, and users benefit from the work
+of a much larger community.
+
+Multi-language component support allows component authors to project their API into all Pulumi-supported languages,
```
Comment (spara) [inline content/blog/multi-lang-components/index.md:38]:
```suggestion
Multi-language component support allows component authors to project their API into all Pulumi-supported languages
```

File: content/blog/multi-lang-components/index.md [hunk 6]
```diff
@@ -0,0 +1,163 @@
...
+idiomatically, and without requiring knowledge of all of these languages. First-class authoring support for multi-language
```
Comment (spara) [inline content/blog/multi-lang-components/index.md:39]:
```suggestion
idiomatically and without requiring knowledge of all of these languages. First-class authoring support for multi-language
```

**Comment by lblackstone** [comment]
@spara Still need a meta image, but I think this is RFR otherwise.

**Review by lblackstone — COMMENTED** [review]
I like the edits! LGTM

**Review by infin8x — APPROVED** [review]
🚀  it :)

**Pull request merged**